benchexec/container.py

```diff
@@ -115,6 +115,12 @@
 )
 """Whether we use generated native code for clone or an unsafe Python fallback"""
 
+_ERROR_MSG_USER_NS_RESTRICTION = (
+    "Ubuntu 24.04 restircts unprivileged user namespaces, please try "
```

Comment:
I reviewed the release notes for Ubuntu 23.10 and found that this version introduced the restriction, but it was not enabled by default. It became the default in version 24.04. Before updating my machine to 24.04, I was using 23.10 and did not encounter any errors caused by the restriction on user namespaces. To confirm when the restriction was introduced as a default option in the Ubuntu kernel, the best approach would be to check on a machine running Ubuntu 23.10, but this is somewhat inconvenient. However, the statement "Ubuntu restricts unprivileged user namespaces since version 23.10" is not incorrect, I will amend this error message.

Reply:
Ah, thanks for checking! I hadn't noticed. Then we can just write "since 24.04".

Reply:
I just forgot to include the link to the release notes. Please check this [link](https://discourse.ubuntu.com/t/mantic-minotaur-release-notes/35534?_gl=1*178yt1l*_gcl_au*MTM0MTg5ODM0MC4xNzE4MTU4NDAz&_ga=2.40433895.455817366.1718158403-1173811805.1718158401#security-improvements-12). :-)

Reply:
Sure, it confirms what you said. But I would prefer the link to the blog post to be used as documentation.